pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -19,7 +19,7 @@
 
 , alsaSupport ? stdenv.isLinux, alsaLib
 , pulseaudioSupport ? stdenv.isLinux, libpulseaudio
-, ffmpegSupport ? true, gstreamer, gst-plugins-base
+, ffmpegSupport ? true
```

Comment:
Shouldn't this flag be removed entirely? I am wondering because how is `ffmpeg` pulled in if enabled?

The tor-browser-bundle has an explicit reference to ffmpeg.

Reply:
AFAICT it bundles the needed headers and the libraries are loaded at runtime via `dlopen` so the package isn't needed at build time. For runtime the libraries are added to the `LD_LIBRARY_PATH` in the wrapper (`wrapper.nix` next to this file) based on whether the browser that is being wrapped was built with ffmpeg support.